content/foundations/getting-started.mdx

```diff
@@ -0,0 +1,74 @@
+</Box>
+
+## A cohesive experience
+
+GitHub has a global audience. Millions of people use GitHub every day for many different reasons, in many different ways. They all come from different backgrounds. We can’t infer their physical or cognitive abilities, or devices they use, or the internet connection they have. All of that needs to be taken into account before, during, and after the design process.
+
+Familiar patterns help people intuitively navigate to the information they're looking for, while also helping people discover new features. If pages and features don’t share the same metaphors, people get confused more easily, it makes it harder for them to understand the features we provide.
+
+Primer focuses on the user experience so people can benefit from using a platform that is cohesive, no matter where in the platform they are. If a page behaves and feels familiar, it allows people to focus on the task at hand, instead of requiring them to parse UI that looks completely new.
+
+## Inclusive and accessible design
+
+GitHub is a home for all developers. To be inclusive means we must consider accessibility at the core of how we design.
+
+Accessibility and inclusive design needs to be expected from the start of a project, because it deeply affects its design, how the product behaves, and how it’s built.
+
+Adding accessibility support to a design after the fact generally turns into a difficult retrofit. To make GitHub fully available for everyone, accessibility requirements need to be taken into account during ideation, design, and implementation phases.
+
+For more information, see [Accessibility at GitHub](/accessibility/accessibility-at-github).
+
+## Responsive design
+
+Browsers are found in devices with all different types of shapes and sizes, and we can’t assume a desktop-only experience is enough. Supporting responsive experiences is an essential part of developing for the Web today. In fact, making sure pages work well [on most screen sizes or zoom levels](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) is an accessibility requirement as well.
+
+Nielsen Norman Group [defines Responsive Web Design](https://www.nngroup.com/articles/responsive-web-design-definition/) as “a web development approach that creates dynamic changes to the appearance of a website, depending on the screen size and orientation of the device being used to view it”.
+
+“Adaptive design”, on the other hand, has been a term historically used to identify an approach where entirely separated fixed layouts were created to support mobile devices (MDN has good documentation [on how we got here](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design#historic_website_layouts)).
```

Comment:
Is this important or could we maybe remove this paragraph to shorten the text a bit?

Reply:
@lukasoppermann I think this is an important consideration to clear any possible confusion about our "responsive design" approach being "adaptive". I agree it might be a little confusing right now when thinking about the granularity of the other information — in the future we may move this to a more specific "Responsive design" dedicated page.

For now I think we can treat this information structure iteratively :v: